### PR TITLE
Visual Moderator Dashboard

### PR DESCRIPTION
**Pull request by Sonjajo**
Updated Moderator Dash with clickable apps on the left that expand to the right; added some new sample-data. 
Used the material-UI card component for the left and right cards. CSS for cardDetail on the right can still use some work, will fix this on this week's sprint when I make the form.

## Related PRs

_Optional - any related PRs you're waiting on, or PRs that will conflict, etc_

## Migrations

_Optional - if you added anything to the database through migration(s)_

## Screenshots

_Required for visual changes_
image

CC: @kwfk

### CODE REVIEW

File: pages/moderator/index.tsx [hunk 1]
```diff
@@ -1,30 +1,60 @@
...
-    <h1>Moderator Dash</h1>
-    <List items={items} />
-    <p>
-      <Link href="/">
-        <a>Go home</a>
-      </Link>
-    </p>
-  </Layout>
-);
+const ModList: React.FunctionComponent<Props> = ({ items }) => {
```
Comment (kwfk) [inline pages/moderator/index.tsx]: Let's name this ModeratorDashboard

File: pages/moderator/index.tsx [hunk 2]
```diff
@@ -1,30 +1,60 @@
...
+        <div className={styles.leftCol}>
+          {items.map((a) => (
+            <div key={a.id} onClick={() => clickCard(a)}>
+              <OrgCard items={a} />
+            </div>
+          ))}
+        </div>
+        <div className={styles.rightCol}>
+          <div className={styles.expandedApp}>
+            {' '}
```
Comment (kwfk) [inline pages/moderator/index.tsx]: delete spaces

File: pages/moderator/index.tsx [hunk 3]
```diff
@@ -1,30 +1,60 @@
...
+          </div>
+          <div className={styles.acceptDecline}>
+            <button type="button" className={styles.accept}>
+              {' '}
+              Accept
+            </button>
+            <button type="button" className={styles.decline}>
+              {' '}
+              Decline
+            </button>
```
Comment (kwfk) [inline pages/moderator/index.tsx]: let's use filled MaterialUI buttons with primary color for accept and secondary for decline

File: components/OrgCard.tsx [hunk 4]
```diff
@@ -1,24 +1,30 @@
...
-        <div className={styles.name}>{data.name}</div>
-        {data.date}
-      </a>
-    </Link>
-    <div className={styles.description}>{data.description}</div>
-  </div>
-);
+const OrgCard: React.FunctionComponent<CardProps> = ({ items }) => {
+  return (
+    <Card className={styles.root}>
```
Comment (kwfk) [inline components/OrgCard.tsx]: Add `CardActionArea` to make the card seem clickable
https://material-ui.com/components/cards/#media

File: pages/moderator/index.tsx [hunk 5]
```diff
@@ -1,30 +1,60 @@
...
+  const clickCard = (newCard: OrgApp): void => {
+    setCard(newCard);
+  };
+
+  return (
+    <Layout title="Moderator Dash">
+      <h1>Moderator Dash</h1>
+      <div className={styles.root}>
+        <div className={styles.leftCol}>
+          {items.map((a) => (
```
Comment (kwfk) [inline pages/moderator/index.tsx]: replace `a` with something more descriptive

File: components/OrgCard.tsx (hunk 4 above)
Comment (kwfk) [reply components/OrgCard.tsx]: You may need to fix the styling because it move the image to the top for some reason

**Review by kwfk — CHANGES_REQUESTED** [review]
nice work!

File: components/OrgCard.tsx (hunk 4 above)
Comment (Sonjajo) [reply components/OrgCard.tsx]: done, and fixed!

**Review by kwfk — APPROVED** [review]
I made some refactoring changes so take a look at those but otherwise LGTM

**Comment by kwfk** [comment]
actually @calblueprint/nbjc everyone should read this

**Pull request merged**